tests/Integrations/Laravel/V8_x/RouteCachingTest.php

```diff
@@ -136,12 +136,12 @@ public function testRouteNameNormalization()
     private function routeCache()
     {
         $appRoot = \dirname(\dirname(self::getAppIndexScript()));
-        `cd $appRoot && php artisan route:cache`;
+        `cd $appRoot && DD_TRACE_CLI_ENABLED=0 php artisan route:cache`;
```

Comment:
Why are we disabling tracing (and debug logs) here?

Reply:
Because the path to the integrations loader isn't specified here, so anything loaded by integrations.c will fail in `Error loading deferred integration 'DDTrace\Integrations\Laravel\LaravelIntegration' from DDTrace\Integrations\load_deferred_integration`.

And it's just noise, it's not part of what we test directly.

Reply:
ack